enterprise/mqtt/src/test/java/io/crate/mqtt/operations/MqttIntegrationTest.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * This file is part of a module with proprietary Enterprise Features.
+ *
+ * Licensed to Crate.io Inc. ("Crate.io") under one or more contributor
+ * license agreements.  See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ *
+ * Unauthorized copying of this file, via any medium is strictly prohibited.
+ *
+ * To use this file, Crate.io must have given you permission to enable and
+ * use such Enterprise Features and you must have a valid Enterprise or
+ * Subscription Agreement with Crate.io.  If you enable or use the Enterprise
+ * Features, you represent and warrant that you have a valid Enterprise or
+ * Subscription Agreement with Crate.io.  Your use of the Enterprise Features
+ * if governed by the terms and conditions of your Enterprise or Subscription
+ * Agreement with Crate.io.
+ */
+
+package io.crate.mqtt.operations;
+
+import io.crate.integrationtests.SQLTransportIntegrationTest;
+import io.crate.mqtt.netty.Client;
+import io.crate.settings.SharedSettings;
+import org.elasticsearch.common.settings.Settings;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.BeforeClass;
+
+import static io.crate.mqtt.netty.Netty4MqttServerTransport.MQTT_ENABLED_SETTING;
+import static io.crate.mqtt.netty.Netty4MqttServerTransport.MQTT_PORT_SETTING;
+
+public abstract class MqttIntegrationTest extends SQLTransportIntegrationTest {
+
+    static final int MQTT_PORT = 11883;
```

Comment:
Could we make the port dynamic to support parallel test runs?

Reply:
You mean a port range? Or configurable port via vm arg?

Reply:
Port range. To reproduce flakyness it's very nice if you can launch a test-class multiple times. That doesn't work if they always bind to the same port.